build.gradle

```diff
@@ -14,6 +14,14 @@ plugins {
     id 'io.freefair.lombok' version '5.3.0'
 }
 
+ext {
+    gitBranch = 'git rev-parse --abbrev-ref HEAD'.execute().text.trim()
+}
+
+if (gitBranch != 'main' && gitBranch != 'master' && ! (gitBranch ==~ '(release|hotfix|support)/.*')) {
```

Comment:
I don't remember whether that has already been asked: couldn't that branch list be put elsewhere so that it could be easily shared between our projects?
At least, it could be in `gradle.properties` so that we separate data and code :thinking:

Reply:
`gradle.properties` is a good place for static values.
Here, we have a mix of static values and regular expressions.
It is a bit tricky to write this in `gradle.properties`.
The best way to follow your suggestion would be to share code in a gradle plugin, but it is a step we don't want to take yet.
(do we write a plugin ? How to manage a public plugin ? ...)
This explains the multiple PRs and copies for the moment.

Reply:
Alright, thanks for the inputs! :)